android/src/main/java/com/horcrux/svg/EllipseView.java

```diff
@@ -36,24 +36,44 @@ public void setCx(Dynamic cx) {
         invalidate();
     }
 
+  public void setCx(String cx) {
+    mCx = SVGLength.from(cx);
+    invalidate();
+  }
+
```

Comment:
I think I remember you explaining problems with `Dynamic` and CodeGen. If that's the reason behind every setter being duplicated for `String` and `Dynamic` it would be great to have a comment explaining it somewhere.

Reply:
Good point. I will add a comment in the main PR's description.